### DISCUSSION TITLE
Weird korean tokenizer behavior

### DISCUSSION THREAD

**Discussion by intoxicated** [meilisearch/product#623] [Feedback & Feature Proposal]
![122](https://user-images.githubusercontent.com/848531/221562008-e5cb8beb-1157-4af4-93ff-e90c2869ec97.png)
![111](https://user-images.githubusercontent.com/848531/221562016-19c0be7c-5a2c-480a-94c1-aea60189f279.png)

 is this normal behavior for korean tokenizer? 

감 shows two results and 감독 shows one result where previous two records contain 감독, (highlighting is also inaccurate)

**Comment by ManyTheFish** [comment]
Hello @intoxicated,

Thank you for the report, about this highlighting issue, could you retry with version [v1.0.2](https://github.com/meilisearch/meilisearch/releases/tag/v1.0.2)? This should be fixed. If it's not the case, don't hesitate to open a [bug report issue directly on Meilisearch](https://github.com/meilisearch/meilisearch/issues/new/choose) pinging me.

Regarding Korean support, we recently changed how we segment words in this PR: https://github.com/meilisearch/charabia/pull/154.
We have a dedicated discussion for Korean Language support: https://github.com/meilisearch/product/discussions/564,
As a Korean speaker, you may help me target this change's limits and suggest modifications to enhance the Korean Language support!

I lock this conversation in favor of the dedicated discussion below:
https://github.com/meilisearch/product/discussions/564

Thanks again!
